scripts/dockerfiles/Dockerfile.build

```diff
@@ -11,7 +11,8 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-FROM golang:1.12
+ARG GO_VERSION=1.15
```

Comment:
I would like to know what others think, but I'd rather not use a default value so that if our main makefile is wrong, the rest of the make files would fail, instead of silently using go 1.15.